packages/itwin/grouping-mapping-widget/src/widget/components/HorizontalTile.tsx

```diff
@@ -6,23 +6,30 @@ import type { ReactNode } from "react";
 import React from "react";
 import { Text } from "@itwin/itwinui-react";
 import "./HorizontalTile.scss";
+import classNames from "classnames";
 
-interface HorizontalTileProps {
+export interface HorizontalTileProps {
   title: string;
   actionGroup: ReactNode;
   subText?: string;
+  onClick?: () => void;
   onClickTitle?: () => void;
   titleTooltip?: string;
   subtextToolTip?: string;
+  selected?: boolean;
+  dragHandle?: ReactNode;
 }
 
-export const HorizontalTile = ({ title, subText, onClickTitle, titleTooltip, subtextToolTip, actionGroup}: HorizontalTileProps) => {
+export const HorizontalTile = ({ title, subText, onClick, onClickTitle, titleTooltip, subtextToolTip, actionGroup, selected, dragHandle}: HorizontalTileProps) => {
```

Comment:
I don't think destructuring helps when you have that many properties. I think simply using `props.onClick` would be good. Now the props do not fit into the screen.

Reply:
done